src/AppInstallerCLICore/Workflows/GetRequestedUrlFromManifest.cpp

```diff
@@ -0,0 +1,56 @@
+#include "pch.h"
+#include "GetRequestedUrlFromManifest.h"
+
+#include "ManifestComparator.h"
+#include "ShellExecuteInstallerHandler.h"
+#include "WorkflowBase.h"
+
+using namespace AppInstaller::CLI;
+using namespace AppInstaller::Utility;
+using namespace AppInstaller::Manifest;
+
+namespace AppInstaller::CLI::Workflow
+{
+    void GetRequestedUrlFromManifest(Execution::Context& context)
+    {
+        const auto& manifest = context.Get<Execution::Data::Manifest>();
+        Workflow::ManifestComparator manifestComparator(context.Args);
+        const auto selectedLocalization = manifestComparator.GetPreferredLocalization(manifest);
+        if (context.Args.Contains(Execution::Args::Type::Homepage)) {
```

Comment:
> { [](start = 67, length = 2)

More style things, we use braces on new lines except in rare one line cases:

if (condition)
{
}